### DISCUSSION TITLE
Determining percentage of outliers for RANSAC

### DISCUSSION THREAD

**Discussion by rockyx50** [chulminy/CIVE497-CIVE700#131] [General]
In the lecture, we were taught how to determine the number of iterations (N) the RANSAC algorithm needs to run in order to get an accurate model estimation (P = 99%). This iteration count is dependent on the the percentage of outliers we have in our data (epsilon) and the more outliers we have in our data the more iterations are required. How are you supposed to determine the percentage of outliers you have in your data?

**Comment by chulminy** [comment]
This is a very good question. A short answer is we do not know the actual ratio. We could empirically obtain the outlier ratio.

If you are not able to test it to obtain this number and assure that it's more than 0.5, you can simply set 0.5. 
In fact, there is no rule but you can set a more conservative value. As you set lower outlier ratio, you should increase N but likely get an inlier set.
